.buildkite/scripts/common.sh

```diff
@@ -69,3 +71,32 @@ with_Terraform() {
     chmod +x ${WORKSPACE}/terraform
     terraform version
 }
+
+google_cloud_auth() {
+    secretFileLocation=$(mktemp -d -p . -t "${TMP_FOLDER_TEMPLATE_BASE}.XXXXXXXXX")/google-cloud-credentials.json
```

Comment:
Where is this stored at $(pwd) or ${WORKSPACE} ?

Reply:
fixed